workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/CloudProvisioning/components/UninstallModal.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton from '@clayui/button';
+import {useModal} from '@clayui/modal';
+
+import Modal from '../../../../../../../components/Modal';
+import i18n from '../../../../../../../i18n';
+import useProvisioningData from '../hooks/useProvisioningData';
+
+type OrderItem = ReturnType<
+	typeof useProvisioningData
+>['provisioningTableData'][0];
```

Comment:
Move type inside the hook and rename 🥺 

```suggestion
import useProvisioningData, {ProvisioningRow} from '../hooks/useProvisioningData';
```